src/User.js

```diff
@@ -12,7 +12,8 @@ class User {
   }
 
   returnFirstName() {
-    let splitName = user.name.split(" ");
+    let splitName = currentUser.name.split(" ");
+    console.log(splitName)
     return splitName[0];
```

Comment:
Good job recognizing that this needed to be current user, we need to remember to go back and take out that console log